### ISSUE TITLE
Enable Contexts in Home by default

### ISSUE THREAD

**Issue by tdipisa** [geosolutions-it/MapStore2#7861]
## Description

From 2022.01.00 The Context tab in Home should be enabled by default in MapStore.

**What kind of improvement you want to add?** (check one with "x", remove the others)
- [X] Minor changes to existing features
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**Comment by offtherailz** [comment]
What about duplication between context-manager and home? We should keep it for the moment or should not?

**Comment by tdipisa** [comment]
Yes, of course, but we can also consider to add in Home another button available only to admin user (at least by default) to create new contexts and remove the admin section. We need app contexts to be more visible and navigable.
